subworkflows/local/visualization.nf

```diff
@@ -0,0 +1,56 @@
+//
+// Create Krona visualizations
+//
+
+include { KRAKENTOOLS_KREPORT2KRONA } from '../../modules/nf-core/modules/krakentools/kreport2krona/main'
+include { KRONA_CLEANUP } from '../../modules/local/krona_cleanup'
+include { KRONA_KTIMPORTTEXT } from '../../modules/nf-core/modules/krona/ktimporttext/main'
+
+
+workflow VISUALIZATION {
+    take:
+    profiles
+
+    main:
+    ch_krona_text = Channel.empty()
+    ch_krona_html = Channel.empty()
+    ch_versions = Channel.empty()
+
+    /*
+        Split profile results based on tool they come from
+    */
+    ch_input_profiles = profiles
+        .branch {
+            kraken2: it[0]['tool'] == 'kraken2'
+            unknown: true
+        }
+
+    /*
+        Convert Kraken2 formatted reports into Krona text files
+    */
+    ch_kraken_reports = ch_input_profiles.kraken2
+    KRAKENTOOLS_KREPORT2KRONA ( ch_kraken_reports )
```

Comment:
```suggestion
    KRAKENTOOLS_KREPORT2KRONA ( ch_input_profiles.kraken2 )
```

Reply:
This is done this way so I can add centrifuge next